server/lib/nl/fulfillment/triple.py

```diff
@@ -37,15 +73,11 @@ def populate(uttr: nl_uttr.Utterance) -> bool:
     if '->' in prop or '<-' in prop:
       uttr.counters.err('triple_property_unsupported', prop)
       continue
-    entity_dcids = [e.dcid for e in uttr.entities]
-    prop_values = raw_property_values(entity_dcids, prop)
-    if not any(prop_values.values()):
-      uttr.counters.err('triple_property_failed_existence', prop)
-      continue
-    state = PopulateState(uttr=uttr)
-    chart_vars = ChartVars(props=[prop])
-    return add_chart_to_utterance(ChartType.ANSWER_WITH_ENTITY_OVERVIEW,
-                                  state,
-                                  chart_vars, [],
-                                  entities=uttr.entities)
+    else:
+      if _populate_any_direction_prop(uttr, prop):
+        # Currently only handling one prop, so if populate succeeds for any one
+        # prop, return.
+        return True
```

Comment:
Any specific reason why that is?

Reply:
no specific reason, but easiest to handle for now & from the mocks, it looks like we're trying to answer the user's question with a single answer OR if user asks for multiple props in the query, then we show the table, but I think that would be similar to a multi-var case

Reply:
I see.  Have a TODO to revisit this?  Because I think for the general case matching multiple props could be useful.

Reply:
done